screed/screedRecord.py

```diff
@@ -129,6 +112,18 @@ def __ne__(self, given):
         except AttributeError:
             raise TypeError("Cannot compare to given type: %s" % type(given))
 
+    def __lt__(self, given):
+        raise NotImplementedError
```

Comment:
According to https://docs.python.org/2/reference/datamodel.html#object.__lt__ you should be returning the singleton type `NotImplemented` when dealing with rich comparators. Raising `NotImplementedError` would be correct in all other cases. Likewise for the three methods below.